public/javascript/main.js

```diff
@@ -377,4 +377,16 @@ $(function(){
           $(this).data('chart', chart);
       }
   })
+
+  // Event tracking
+  // Clicks on elements with data-ga-track-click attribute will be recorded
+  function trackClick(eventDescription){
+    eventDescription = eventDescription || 'Undescribed event.';
+    ga('send', 'event', 'user interaction', eventDescription, document.title);
```

Comment:
Probably best to force the Analytics call to be synchronous (ie: give it a `hitCallback` parameter) and then perform the default browser action (eg: navigating to the next page) only once we know the Analytics call has either completed or failed.

https://github.com/mysociety/fca-investment-tool/blob/master/web/public/js/analytics.js (mysoc team login required) is a pretty bulletproof wrapper we made previously, that uses jQuery deferred objects to keep the callback chaining super obvious.